test/index.js

```diff
@@ -60,23 +60,38 @@ describe('Schwifty', () => {
         useNullAsDefault: true
     };
 
-    const getServer = (options, cb) => {
+    async function getServer (options) {
 
         const server = new Hapi.Server();
-        server.connection();
 
-        server.register({
-            register: Schwifty,
-            options
-        }, (err) => {
+        try {
 
-            if (err) {
-                return cb(err);
-            }
+            await server.register({
+                plugin: Schwifty,
+                options
+            });
 
-            return cb(null, server);
-        });
-    };
+            return server;
+        } catch (err) {
+
+            console.log("Initial registration failed", err);
+            throw err;
+        }
+
+    }
+
+    async function initializeServer (server) {
+
+        try {
+
+            await server.initialize();
+            return server;
+        } catch (error) {
```

Comment:
The linter will catch this– typically `catch()` will go on the next line in our style.

Reply:
ah, gotcha. thanks for the note. I'm not sure I'd seen that from the linter...hm.